src/pages/Home.js

```diff
@@ -141,7 +154,19 @@ class Home extends Component {
               ])
             ])
           ]),
-          div({ className: 'row', style: { background: '#eff0f2', margin: '50px 0', padding: '60px 0 72px 0' } }, [
+          div({ className: 'row', style: { background: '#eff0f2', margin: '50px 0', padding: '48px 0 60px 0' } }, [
+            div({ className: 'row', style: { background: '#eff0f2', margin: '0 0 48px 0'} }, [
+              div({ className:'col-lg-6 col-md-6', style: { display: 'flex', justifyContent: 'center' }}, [
+                button({ className: 'btn-primary', style: buttonStyleBig }, [
+                  a({href: 'https://databiosphere.github.io/duos-ui/DatasetRegistrationInfo', style: {color: '#fff'}}, ['Register a dataset in DUOS'])
```

Comment:
Are we going to be linking to the github pages instead of creating new pages inside duos-ui for future FAQ docs?

Reply:
I'm not sure about future documents, but as of right now I do believe we are linking the github pages.